python-sdk/nuscenes/eval/detection/tests/test_loader.py

```diff
@@ -0,0 +1,132 @@
+# nuScenes dev-kit.
+# Code written by Sourabh Vora, 2019.
+# Licensed under the Creative Commons [see licence.txt]
+
+import os
+import unittest
+
+from nuscenes import NuScenes
+from nuscenes.eval.detection.loaders import filter_eval_boxes
+from nuscenes.eval.detection.data_classes import EvalBox, EvalBoxes
+
+
+class TestLoader(unittest.TestCase):
+    def test_filter_eval_boxes(self):
+        """
+        This tests runs the evaluation for an arbitrary random set of predictions.
+        This score is then captured in this very test such that if we change the eval code,
+        this test will trigger if the results changed.
+        """
+        assert 'NUSCENES' in os.environ, 'Set NUSCENES env. variable to enable tests.'
+
+        nusc = NuScenes(version='v1.0-mini', dataroot=os.environ['NUSCENES'], verbose=False)
+
+        sample_token = '0af0feb5b1394b928dd13d648de898f5'
+        # This sample has a bike rack instance 'bfe685042aa34ab7b2b2f24ee0f1645f' with these parameters
+        # 'translation': [683.681, 1592.002, 0.809],
+        # 'size': [1.641, 14.465, 1.4],
+        # 'rotation': [0.3473693995546558, 0.0, 0.0, 0.9377283723195315]
+
+        max_dist = {'car': 50,
```

Comment:
You could get this from the config factory in `config.py` also if you like.

Reply:
Done. Thanks!